README.md

```diff
@@ -13,7 +13,8 @@ designed to work well with [Bazel](http://bazel.io).
 git_repository(
     name = "subpar",
     remote = "https://github.com/google/subpar",
-    commit = "HEAD",
+    tag = "1.0",
+    sha256 = "Fill in sha256 here",
```

Comment:
You can specify exactly one of ['tag', 'commit'] and optionally 'sha256', which is checked by Bazel not git.  And 'commit' = 'HEAD' isn't accepted by Bazel anymore.

https://bazel.build/versions/master/docs/be/workspace.html#git_repository

    git_repository(name, commit, init_submodules, remote, sha256, tag)